runtime/src/bank_forks.rs

```diff
@@ -268,7 +268,7 @@ impl BankForks {
         if entry.get().is_empty() {
             entry.remove_entry();
         }
-        Some(bank.clone_without_scheduler())
```

Comment:
fun fact: this is the single actual code change, excluding all the type, comment, test code.

Reply:
Got it. So instead of dropping the scheduler part of the bank, we are just taking ownership of the bank with its scheduler.